app/main/forms/brief_forms.py

```diff
@@ -0,0 +1,91 @@
+from flask_wtf import Form
+from wtforms import IntegerField, SelectMultipleField
+from wtforms.validators import NumberRange
+
+
+class BriefSearchForm(Form):
```

Comment:
> Setting that aside, I'd rather we have one approach to filter implementation than two. It feels like adding lot and status questions to the frameworks is not really a blocker, since we've done this for G-Cloud before. And the argument for extending it to filter on other brief fields is less abstract than it is with wtforms – we have a working example.

(looking further into this now...)